packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -511,49 +512,81 @@ handler.scanForMissingPullRequests = async function scanForMissingPullRequests(
  * @returns void
  */
 function handler(app: Probot) {
-  //meta-note about the schedule.repository as any; currently GH does not support this type, see
-  //open issue for a fix: https://github.com/octokit/webhooks.js/issues/277
-  app.on('schedule.repository' as '*', async context => {
-    if (context.payload.syncLabels === true) {
-      const owner = context.payload.organization.login;
-      const repo = context.payload.repository.name;
-      await syncLabels(context.octokit, owner, repo, MERGE_ON_GREEN_LABELS);
+  // This scheduled job iterates through the PR database and removes PRs
+  // That are closed or do not have an applicable label anymore.
+  app.on('schedule.global' as '*', async context => {
+    if (context.payload.cleanUp !== true) {
```

Comment:
So, switching the logic this way will make it so that the regular cron jobs (not the clean-up ones) will not execute. This isn't equivalent to 
`if (context.payload.cleanUp === true)`

Reply:
It's a guard clause in case we have multiple `global` cron events in the future. The other non-global crons won't attempt to run this block.

Reply:
The regular merge ones are now on `cron_type: installation` which runs in a `schedule.installation` handler.